src/schema.ts

```diff
@@ -43,7 +41,6 @@ export function createSchema<TApi, TData>(services: Array<Service<TApi, TData>>)
       Query: {},
       Mutation: {},
       Subscription: {},
-      Upload: GraphQLUpload,

```

Comment:
But we need to keep this, too - or not?

Reply:
it looks like it is not necessary and configured automatically (reference https://github.com/apollographql/apollo-server/blob/54ff1e68b62f9fbdc366f48562caf531cadf6abb/packages/apollo-server-core/src/ApolloServer.ts#L482-L501)